From b05b2826365a7dbc6ca1bf0977b848055cd0cbb6 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Sun, 13 Mar 2022 20:26:05 -0400
Subject: [PATCH] meson: fix misspelled kwarg name

set10 doesn't have a `Description` kwarg, it does have a `description`
kwarg though.

This caused the conf variable to not have a description when it should
have one, and newer versions of Meson with better argument validation
error out with:

meson.build:58:5: ERROR: configuration_data.set10 got unknown keyword arguments "Description"
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 55054f441..8c93be9d1 100644
--- a/meson.build
+++ b/meson.build
@@ -55,7 +55,7 @@ lib_name    = meson.project_name() + '-' + api_version
 stability_version_number = (major_version != 0 ? minor_version : micro_version)
 stable = (stability_version_number % 2 == 0)
 
-conf.set10('BABL_UNSTABLE', not stable, Description:
+conf.set10('BABL_UNSTABLE', not stable, description:
   'Define to 1 if this is an unstable version of BABL.')
 
 conf.set       ('BABL_MAJOR_VERSION',    '@0@'.format(major_version))
-- 
GitLab

